src/components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+function Button({ text, color, backgroundColor, width, height, fontSize }) {
+  return (
+    <button
+      style={{
+        color: color,
+        backgroundColor: backgroundColor,
+        width: width,
+        height: height,
+        border: "none",
+        borderRadius: "50px",
```

Comment:
I prefer to pass a class name props or object of styles instead of pass each prop